core/storage/question/gae_models.py

```diff
@@ -49,6 +51,17 @@ class QuestionModel(base_models.VersionedModel):
         ndb.IntegerProperty(required=True, indexed=True))
     # The ISO 639-1 code for the language this question is written in.
     language_code = ndb.StringProperty(required=True, indexed=True)
+    # The status of the question. It can be 'private', 'approved',
```

Comment:
Shouldn't be here. Rights changes shouldn't be part of question data version history.

Reply:
I've removed status and it functionalities from this PR